File: python/Program/produk.py
class Produk:
    # Constructor
    def __init__(self):
        self.__id = ""
        self.__nama = ""
        self.__harga = 0
        self.__stok = 0

    # Getter 
    def get_id(self):
        return self.__id

    def set_id(self, id):
        self.__id = id

    def get_nama(self):
        return self.__nama

    def set_nama(self, nama):
        self.__nama = nama

    # Setter
    def get_harga(self):
        return self.__harga

    def set_harga(self, harga):
        self.__harga = harga

    def get_stok(self):
        return self.__stok

    def set_stok(self, stok):
        self.__stok = stok

File: php/Program/class/Produk.php
<?php
class Produk {
    // atribut class produk
    private $id;
    private $nama;
    private $harga;
    private $stok;
    private $gambar; // gambar

    // Constructor
    public function __construct($id = "", $nama = "", $harga = 0, $stok = 0, $gambar = "") {
        $this->id = $id;
        $this->nama = $nama;
        $this->harga = $harga;
        $this->stok = $stok;
        $this->gambar = $gambar;
    }

    // Getter
    public function getId() { return $this->id; }
    public function getNama() { return $this->nama; }
    public function getHarga() { return $this->harga; }
    public function getStok() { return $this->stok; }
    public function getGambar() { return $this->gambar; }

    // Setter
    public function setId($id) { $this->id = $id; }
    public function setNama($nama) { $this->nama = $nama; }
    public function setHarga($harga) { $this->harga = $harga; }
    public function setStok($stok) { $this->stok = $stok; }
    public function setGambar($gambar) { $this->gambar = $gambar; }
}
